scripts/install_cairo_bindings.sh

```diff
@@ -5,37 +5,33 @@ set -e
 # clean up
 if [ "$1" == "--cleanup" ]; then
   poetry env info -p | xargs rm -rf
-  if [[ $(uname -m) == 'arm64' ]]; then
+  if [[ $(uname -s) == 'Darwin' ]]; then
     CFLAGS=-I/opt/homebrew/opt/gmp/include LDFLAGS=-L/opt/homebrew/opt/gmp/lib poetry install
   else
     poetry install
   fi
 fi
 
 function install_dev() {
-  git pull --recurse-submodules
+  git pull origin $(git rev-parse --abbrev-ref HEAD) --ff-only --recurse-submodules
```

Comment:
Could you please explain why we need to do it like this?

Reply:
We need to be able to build from users branches, in order to provide test and pre releases. 
The `git rev-parse --abbrev-ref HEAD` gets the current branch name.